presto-spi/src/main/java/io/prestosql/spi/type/LongTimestamp.java

```diff
@@ -74,4 +75,13 @@ public int compareTo(LongTimestamp other)
         }
         return Integer.compareUnsigned(picosOfMicro, other.picosOfMicro);
     }
+
+    @Override
+    public String toString()
+    {
+        return new StringJoiner(", ", LongTimestamp.class.getSimpleName() + "[", "]")
```

Comment:
Would it be better to use a human readable format? i.e.
```java
return formatTimestamp(MAX_PRECISION, epochMicros, picosOfMicro);

Reply:
It can go either way. For debugging purposes, I've often found it useful to see what the component parts are.

Reply:
> It can go either way. 

True.

I think human-readable should be fine. picos are visible in human; epoch micros -- you would need to expand the object in a debugger.